repo2docker/buildpacks/r.py

```diff
@@ -186,6 +204,13 @@ def get_build_scripts(self):
                 """.format(
                     devtools_version=devtools_version,
                     irkernel_version=irkernel_version
+            ),
+            (
+                "${NB_USER}",
+                # Install shiny library
+                r"""
+                R --quiet -e "install.packages('shiny', repos='https://mran.microsoft.com/snapshot/{}', method='libcurl')"
+                """.format(self.checkpoint_date.isoformat())
```

Comment:
I think this is the offending line -> delete the last closing `)`

Reply:
Made a PR to your repo https://github.com/ryanlovett/repo2docker/pull/1 (I thought these days repo maintainers can push to the PR branches of others?)